ombpdf/annotators.py

```diff
@@ -5,10 +5,10 @@
     ('page_numbers', 'ombpdf.pagenumbers.annotate_page_numbers'),
     ('footnotes', 'ombpdf.footnotes.annotate_footnotes'),
     ('footnote_citations', 'ombpdf.footnotes.annotate_citations'),
-    ('paragraphs', 'ombpdf.paragraphs.annotate_paragraphs'),
     ('underlines', 'ombpdf.underlines.set_underlines'),
     ('lists', 'ombpdf.lists.annotate_lists'),
     ('headings', 'ombpdf.headings.annotate_headings'),
+    ('paragraphs', 'ombpdf.paragraphs.annotate_paragraphs'),
```

Comment:
Moving paragraphs to the end means that any of our other annotations can be used as a signal to break one big paragraph into two.  For example, in M-15-17, there's a header called "Multi-agency native youth priorities" that can be used to do this, I think.  It's particularly useful because pdfminer doesn't detect a blank line under the heading.

Um I might need to revisit this actually and see if this change is absolutely required, but it does generally *seem* like a decent idea.

Reply:
Update: Yes, this is required, I added a test to make sure it doesn't regress in da992f2.